index.html

```diff
@@ -123,12 +120,357 @@ <h1 class="primary-text-1">ľm Mariane
         </div>
 
 
-    </div>
+
 
     <div id="header-background-mobile">
     </div>
 
 
 
+
+    <!-- Works section -->
+    <div id="works-section">
+
+      <div id="align-section">
+
+        <div id="works-section-div1" class="works-section-class">
+
+          <div id="snapshot-portfolio-1" class="snapshot-portfolio-class">
+          </div>
+
+          <div id="block-apart1">
+
+            <div id="block-apart-primary-text-1">
+              <div id="block-apart-primary-text-1-project-title">
+                <h1 id="block-apart-primary-text-1-project-title-text-1" class="block-apart-primary-text">Tonic</h1>
+              </div>
+
+              <div id="frame-2">
+                <div id="client"><p id="client-text" class="style-1-font">CANOPY</p></div>
+                <div id="counter"></div>
+                <div id="role"><p id="role-text" class="style-1-font">Back End Dev</p></div>
+                <div id="counter"></div>
+                <div id="year-label"><p id="year-label-text" class="style-1-font">2015</p></div>
+              </div>
+            </div>
+
+            <div id="block-apart-primary-text-2">
+              <p id="block-apart-primary-text-2-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+
+            <div id="block-apart-tags">
+              <ul id="block-apart-tags-li">
+                <li id="block-apart-tags-item-1" class="block-apart-tags"><a class="block-apart-tags-item">html</a></li>
+                <li id="block-apart-tags-item-2" class="block-apart-tags"><a class="block-apart-tags-item">css</a></li>
+                <li id="block-apart-tags-item-3" class="block-apart-tags"><a class="block-apart-tags-item">javaScript</a></li>
+              </ul>
+            </div>
+
+            <div id="block-apart-action">
+              <button id="block-apart-action-button" type="button">See Project</button>
```

Comment:
- [ ] Consider making sure the buttons follow the button styling: Active, Hover, and Pressed.